cmd/gvisor-website/main.go

```diff
@@ -41,20 +45,28 @@ var redirects = map[string]string{
 }
 
 var prefixHelpers = map[string]string{
-	"cl":     "https://gvisor-review.googlesource.com/c/gvisor/+/%s",
-	"change": "https://gvisor.googlesource.com/gvisor/+/%s",
```

Comment:
Should we change these? If we do, we invalidate all of the existing links. Not the end of the world (there aren't too many), but I was thinking we could leave cl and change and add pr and pull for post-github migration changes. On the other hand, I can see how leaving the obsolete redirect could be confusing.

Reply:
This is a good point. I think we should just add pr/ and pull/.

Reply:
Note that I've added these but the comment does not resolve but the specific lines here did not change.